### PR TITLE
Added batch size and index date to command

### PR DESCRIPTION
**Pull request by cableman**

### CODE REVIEW

File: src/Migrations/Version20211104191514.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20211104191514 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('DROP INDEX is_identifier_idx ON search');
```
Comment (turegjorup) [inline src/Migrations/Version20211104191514.php:18]: Not sure this index was ever deployed to `prod`. Having this `DROP` statement will then cause migration to fail

Comment (cableman) [reply src/Migrations/Version20211104191514.php]: Fixed

**Comment by cableman** [comment]
Don't known what to do with psalm it don't understand symfony input options and I have updated baseline...

**Comment by cableman** [comment]
The psalm errors will be fixed by the symfony plugin in https://github.com/danskernesdigitalebibliotek/ddb-cover-service-importers/pull/91

**Review by turegjorup — APPROVED** [review]

**Pull request merged**